include/openssl/conf.h.in

```diff
@@ -119,7 +119,9 @@ struct conf_st {
     CONF_METHOD *meth;
     void *meth_data;
     LHASH_OF(CONF_VALUE) *data;
-    unsigned int flag_dollarid:1;
+    int flag_dollarid;
+    int flag_abspath;
```

Comment:
why not use the `:1` for both flags?

Reply:
>why not use `:1` for both flags?

Because you cannot pass a bitflag as a parameter.

Reply:
You a pointer to a bitflag. And yes, good point!